src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private int buy = 0;
+    private int supply = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        WorkWithFile workWithFile = new WorkWithFile();
+        String firstResultString = workWithFile.read(fromFileName);
+        String[] allArray = firstResultString.split(",");
+        for (int i = 0; i < allArray.length; i = i + 2) {
+            if (allArray[i].equals("buy")) {
+                buy = buy + Integer.parseInt(allArray[i + 1]);
+            }
+            if (allArray[i].equals("supply")) {
+                supply = supply + Integer.parseInt(allArray[i + 1]);
+            }
+        }
+        int result = supply - buy;
+        String resultString = "supply," + supply + System.lineSeparator() + "buy," + buy
+                + System.lineSeparator() + "result," + result;
+        workWithFile.write(resultString, toFileName);
```

Comment:
```suggestion
        String dataFromFile = read(fromFileName);
        String report = createReport(dataFromFile);
        write(toFileName, report);
```
lets divide logic like this